apps/etherscan/src/app/routes.tsx

```diff
@@ -43,7 +43,7 @@ export const DisplayRoutes = () => (
                 </DefaultLayout>} />
       <Route
         path="/settings"
-        element={<DefaultLayout from="/settings" title="Set Etherscan API Key">
+        element={<DefaultLayout from="/settings" title="Set Explorer API Key">
```

Comment:
if each explorer has a different api key, we should probably allow specifying an api key per explorer.. (